platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_performance.adoc

```diff
@@ -42,3 +42,20 @@ NOTE: Do not rely on the default encoding of the underlying JVM or operating sys
 === Tags
 
 `<<tags_reference.adoc#tag:properties,#properties>>`, `<<tags_reference.adoc#tag:caching,#caching>>`
+
+[[use_build_cache]]
+== Use the Build Cache
```

Comment:
We should have a link on how to turn it on.

Reply:
This PR should have been drafted. It was not ready for review yet.